pkg/kubelet/types/constants.go

```diff
@@ -22,9 +22,12 @@ const (
 
 	// different container runtimes
 	DockerContainerRuntime = "docker"
-	RktContainerRuntime    = "rkt"
 	RemoteContainerRuntime = "remote"
 
+	// rkt is deprecated and it won't really work.
+	// We break execution when it's requested.
```

Comment:
Add a TODO stating when this can be removed

Reply:
I removed the constant (since it's used in a single place) but I left in place the check that breaks and warns to use rktlet instead. I imagine we can leave that check around, it doesn't really hurt a lot... Right?